### DISCUSSION TITLE
opleidingscommissie

### DISCUSSION THREAD

**Discussion by Stefan-Espant** [fdnd/docs.fdnd.nl#73] [Reflectie]
### Formatie OC
In 2022 werden Suus en Stefan onderdeel van de OC. Samen met de docenten Koop en Suus vormen zij de definitieve OC. Hiermee werd de gewenste balans tussen docenten en studenten behouden. De formatie van de OC verliep soepel.

### Deelnemen aan vergaderingen
Omdat de OC nog vrij nieuw was, waren we in het begin zoekende naar onze inspraakmogelijkheden. Gaandeweg kregen we meer duidelijkheid over onze rol, inclusief inspraak op onder andere: de OER, het budget van de opleiding, het deelnemen aan externe OC-vergaderingen en het opzetten van interne vergaderingen.

### Over het algemeen

**Suus**
We hadden regelmatig meetings met de OC. Vaak liepen we inhoudelijk vast op onderwerpen als budgetten en de OER, waardoor er soms weinig concreets uit de vergaderingen kwam.

**Stefan**
Over het algemeen hebben we het proces van de OC goed uitgevoerd. Ik heb actief deelgenomen aan externe vergaderingen, trainingen en interne vergaderingen. Ik denk dat ik onze opleiding tijdens deze momenten goed heb vertegenwoordigd. Daarnaast heb ik me ook ingezet voor het werven van een nieuw OC-lid. Over het algemeen ben ik tevreden over de OC, en ik hoop in de toekomst mijn taak over te dragen aan een andere enthousiaste deelnemer.

**Comment by ju5tu5** [comment]
Opgenomen in de lopende tekst van het document.
